Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apk update && apk add --no-cache \
     curl \
     gcc \
     bash \
-    mailcap
+    mailcap \
+    build-base
```

Comment:
gcc is already installed. Build-base provides gcc, g++, make, musl-dev and binutils. You can remove gcc as it is duplicated by build-base or maybe you don't really need build-base if all you were after was gcc.

Reply:
I reverted this back since now it is cgo-free (https://github.com/argoproj/argo-workflows/pull/12736#discussion_r1519971792)